packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -599,6 +599,7 @@ export type CallCompositeOptions = {
     galleryOptions?: {
         layout?: VideoGalleryLayout;
     };
+    survey?: boolean;
```

Comment:
For consistency around naming, should it be something like `surveyOptions`

Reply:
It's enabling and disabling a feature similar to errorBar, so I followed the naming and named it as survey